security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -0,0 +1,77 @@
+//OpenShift Compliance Operator Release Notes
+[id="compliance-operator-release-notes"]
+= Compliance Operator release notes
+:context: compliance-operator-release-notes-v0
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+[id="compliance-operator-inclusive-language"]
+== Making open source more inclusive
+
+Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[Red Hat CTO Chris Wright's message].
+
+[id="compliance-operator-release-notes-0-1-44"]
+== Release Notes for Compliance Operator 0.1.44
+Issued: 2021-
+
+The Compliance Operator 0.1.44 is now available with enhancements and bug fixes.
+
+[id="compliance-operator-0-1-44-new-features-and-enhancements"]
+=== New features and enhancements
+
+* To control whether scan scheduling is strict or not strict - a new `strictNode` option is available for `ComplianceScan`, `ComplianceSuite` and `ScanSetting` scans. This option defaults to true (strict), meaning that the operator will error out if a scan pod can't be scheduled. Switching the option to true makes the scan more permissive an go forward. Useful for clouds with ephemeral nodes.
++
+* In this release, the `nodeSelector` and `tolerations` in the `ScanSettings` object are now configurable  This enables deployers to configure where the Result Server will run, and thus what node will host the persistent volume that will contain the raw results. This allows us to schedule a pod that makes use of a persistent volume on the master nodes.
++
+* In this release, the Compliance Operator can now remediate `KubeletConfig` objects.
```

Comment:
Do you think we should expand on this feature? Honestly, I think this feature and the variables would benefit from a subsection in the CO docs. We're still working on the upstream documentation as well, here's a draft: https://github.com/Vincent056/compliance-operator/blob/master/doc/kubeletConfig-remediations.md